aws/index.ts

```diff
@@ -63,7 +52,7 @@ new k8s.helm.v3.Chart("akka-operator", {
 
 
 let bootstrapServersSecretName: string | null = null;
-let kafkaCluster: model.KafkaCluster | null = null;
+let kafkaCluster: eks.MskKafkaCluster | null = null;
```

Comment:
Is this limited to MSK Kafka Clusters? If users brought their own, should this variable support definition of non-MSK clusters?

Reply:
I guess we don't need these Pulumi templates to support all possible combinations, just one to get people started. 

LGTM